src/main/java/net/neoforged/neoform/runtime/cli/RunNeoFormCommand.java

```diff
@@ -508,30 +522,27 @@ private static ExecutionNode getOrAddDevTransformsNode(NeoFormEngine engine) {
             }
```

Comment:
Could we somehow inline the rest of this function into a neoform-specific branch? Given that it will never trigger for neoforge.